RPS Project/Game.py

```diff
@@ -0,0 +1,60 @@
+import sys
+import random
+
+score = [0,0,0]
+history = [0,0,0]
+count = [0]
+
+#Friendly greeting
+input("Good Lobning. Press enter.")
+print("Choices: \n 0 = Rock \n 1 = Paper \n 2 = Scissors \n 3= exit")
+
+#Returns game result
```

Comment:
If you are going to ever have functions with non-obvious return values, you must document them. Without looking at your `game()` function or parsing the logic of this function, there's no easy way for a developer to know that this function will return 0/1/2, and that those mean draw/win/lose.